Modules/ExtendedPlayerControl.cs

```diff
@@ -640,40 +640,18 @@ public static void RpcExileV2(this PlayerControl player)
     }
     public static void RpcMurderPlayerV3(this PlayerControl killer, PlayerControl target)
     {
-        if (target.Is(CustomRoles.Pestilence) && killer != target) { killer.SetRealKiller(target); (killer, target) = (target, killer); Main.PlayerStates[target.PlayerId].deathReason = PlayerState.DeathReason.PissedOff; goto postPest; }
-        else if (target.Is(CustomRoles.Pestilence) && target.GetRealKiller() != null && target.GetRealKiller() != target) { var truekill = target.GetRealKiller(); (killer, target) = (target, truekill); Main.PlayerStates[target.PlayerId].deathReason = PlayerState.DeathReason.PissedOff; goto postPest; }
-        else if(target.Is(CustomRoles.Pestilence)) { return; }
-        postPest:
-
-        if (Solsticer.OnCheckRpcMurderv3(killer, target))
-            return;
-
-        if (target.Is(CustomRoles.Susceptible))
-            Susceptible.CallEnabledAndChange(target);
-
-        if (killer.PlayerId == target.PlayerId && killer.shapeshifting)
+        if (killer.PlayerId == target.PlayerId && killer.shapeshifting) 
         {
             _ = new LateTask(() => { killer.RpcMurderPlayer(target, true); }, 1.5f, "Shapeshifting Suicide Delay");
             return;
         }
```

Comment:
was this something important? Should jsut simply be moved to rpcmurder then.

Reply:
No, it's not needed
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/104814436/0588dbaa-4458-48f0-ab85-bf0b6918cfa5)